runtimes/spiritnet/src/lib.rs

```diff
@@ -817,6 +821,7 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 							| delegation::Call::change_deposit_owner { .. }
 					)
 					| RuntimeCall::Democracy(..)
+					// Excludes `DepositStorage`
```

Comment:
I didn't see this for Peregrine? Can we add it?

Reply:
If you mean the comment, it is already present: https://github.com/KILTprotocol/kilt-node/pull/616/files#diff-56f0782a52c09d5ac94e1b327f8afaae9dd45f76ed0c17e75773c74ed433caf8R836. 

Or what do you mean?